extending-valtimo/document/configuring-search-fields.md

```diff
@@ -49,20 +49,24 @@ To retrieve all the search fields created for a certain document definition use
 `GET /api/v1/document-search/{documentDefinitionName}/fields`. This will return an array of search fields.
 
 ## Update existing search field
-An existing search field can be updated with new values by calling the following endpoint:
+
+Existing search fields can be updated with new values by the endpoint below. The endpoint below accepts a list of search
+fields. The ordering in this list is adhered.
 `PUT /api/v1/document-search/{documentDefinitionName}/fields` and providing the following request body:
 
 ```json
-{
-  "key": "key_to_search_on",
-  "path": "/some/json/path/to/key",
-  "datatype": "boolean",
-  "fieldtype": "single",
-  "matchtype": "exact"
-}
+[
+  {
+    "key": "key_to_search_on",
+    "path": "doc:path.to.key",
+    "dataType": "boolean",
+    "fieldType": "single",
+    "matchType": "exact"
+  }
+]
 ```
 
-The datatype, fieldtype and matchtype have specific values that are allowed.
+The dataType, fieldType and matchType have specific values that are allowed.
```

Comment:
`dataType`, `fieldType` and `matchType` only allow for specific values. A list of these allowed values can be found [here](https://github.com/valtimo-platform/valtimo-documentation/pull/168/files#allowed-values).